### PR TITLE
Add Rake task for PR visual regression comparison

### PR DESCRIPTION
**Pull request by fofr**
Compare production and a review app hosted on Heroku. Pass in a PR number to generate a config and run the test.

eg `bundle exec rake wraith:pr[450]`

* Removes unused `update_document_types` task
* Refactors wraith tasks to DRY up

### CODE REVIEW

**Comment by nickcolley** [comment]
Nice! Should we add something to the Readme so people know they can do this?

**Comment by fofr** [comment]
Updated with documentation and a task for comparing master.

**Comment by fofr** [comment]
Found an issue with Wraith, documented here: https://github.com/BBC-News/wraith/issues/536
And avoiding the issue with 37d402b

File: lib/helpers/wraith_config_helper.rb [hunk 1]
```diff
@@ -29,11 +29,13 @@ def build_paths
     if @paths.is_a? Hash
       @paths.keys.each do |key|
         @paths.fetch(key).each_with_index do |path, index|
-          config_paths[path_index(key, index + 1)] = path
+          config_paths[path_index(key, index + 1)] = path unless path_blacklisted?(path)
```
[the remark this answers is not in the document]
Comment (fofr) [reply lib/helpers/wraith_config_helper.rb]: Good call.

**Comment by tijmenb** [comment]
Is this PR still relevant now that there's https://github.com/alphagov/govuk-visual-regression/pull/1?

**Pull request closed without merging**